intellij-ext/modules/products/rider/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    // Java support
+    id("java")
+    // Kotlin support
+    id("org.jetbrains.kotlin.jvm") version "1.8.10"
+    // Gradle IntelliJ Plugin
+    id("org.jetbrains.intellij") version "1.+"
+
+}
+
+tasks {
+    buildSearchableOptions {
+        enabled = false
+    }
+}
+
+intellij {
+    version.set(properties("platformVersion"))
+    type.set("RD")
```

Comment:
I have no idea why this line is required, as nothing similar happens in build files for other projects. It costed me a lot of sanity points.